python/ray/reporter.py

```diff
@@ -107,6 +112,23 @@ def __init__(self, redis_address, redis_password=None):
     def get_cpu_percent():
         return psutil.cpu_percent()
 
+    @staticmethod
+    def get_gpu_usage():
+        if gpustat is not None:
+            gpu_utilizations = []
+            gpus = []
+            try:
+                gpus = gpustat.new_query().gpus
+            except Exception:
+                logger.debug("gpustat failed to retrieve GPU information.")
+            for gpu in gpus:
+                # Note the keys in this dict have periods which throws
+                # off javascript so we change .s to _s
+                gpu_data: Dict[str, Any] = {'_'.join(key.split('.')): val 
```

Comment:
Variable typing is not supported by Python 3.5. We should remove this unfortunately...